src/js/login/containers/Main.jsx

```diff
@@ -75,6 +80,7 @@ class Main extends React.Component {
       });
       const receiver = window.open(`${myLoginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
       receiver.focus();
+      this.getLoginUrl();
```

Comment:
I don't quite understand how login works, tbh. But it looks this PR just moves the `this.serverRequest` into a wrapper function.